src/lyk.h

```diff
@@ -0,0 +1,19 @@
+#include "lobject.h"
+
+#include "lopcodes.h"
+
+#ifdef USE_YK
```

Comment:
Are these `USE_YK` checks required? And if so, should we just guard to whole file in this way?

Reply:
Yeah, that's a good idea I will guard to whole file

Reply:
done :point_right: bf85892b460abc70b21876665293d8557a483dcf